podcast.rss

```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0" 
+  xmlns:googleplay="http://www.google.com/schemas/play-podcasts/1.0" 
+  xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd">
+  <channel>
+    <title>Velicham Qur'an Dars Series</title>
+    <description>
+      <![CDATA[<p>&nbsp;</p><div><div>ജനപ്രിയ മാധ്യമങ്ങളിലൂടെ പൊതു സമൂഹത്തിലേക്ക് ഖുർആൻ വെളിച്ചം പകരുക എന്ന ലക്ഷ്യത്തോടെ മുന്നോട്ട് പോവുന്ന കൂട്ടായ്മയാണ് വെളിച്ചം ഓൺലൈവ്.<div><div>&nbsp;</div><br /><div>ഉസ്താദ് അബ്ദുൽ വാരിസിന്റെ കീഴിൽ, ആയിരക്കണക്കിന് വിദ്യാർത്ഥികൾ പ്രയോജനപ്പെടുത്തി കൊണ്ടിരിക്കുന്ന ഓഡിയോ പഠന ക്ലാസുകളാണ് ഈ പോഡ്കാസ്റ്റ് സീരീസിൽ ഉൾക്കൊള്ളിച്ചിരിക്കുന്നത്.</div></div><p>&nbsp;</p>]]>
+    </description>
+    <language>ml</language>    <!-- two-letter ISO 639-1 code for Malayalam -->
+    <link>https://velicham-onlive.github.io</link>
+    <!-- google play configs -->
+    <googleplay:author>Velicham Onlive</googleplay:author>
+    <googleplay:image href="https://velicham-onlive.github.io/images/icons/ic_velicham.png"/>
+    <googleplay:category text="Religion &amp; Spirituality"/>
+    <!-- itunes configs -->
+    <itunes:author>Velicham Onlive</itunes:author>
+    <itunes:image href="https://velicham-onlive.github.io/images/icons/ic_velicham.png"/>
+    <itunes:category text="Religion &amp; Spirituality" />
+    <itunes:keywords></itunes:keywords>
+    <itunes:summary>
+      <![CDATA[<p>&nbsp;</p><div><div>ജനപ്രിയ മാധ്യമങ്ങളിലൂടെ പൊതു സമൂഹത്തിലേക്ക് ഖുർആൻ വെളിച്ചം പകരുക എന്ന ലക്ഷ്യത്തോടെ മുന്നോട്ട് പോവുന്ന കൂട്ടായ്മയാണ് വെളിച്ചം ഓൺലൈവ്.<div><div>&nbsp;</div><br /><div>ഉസ്താദ് അബ്ദുൽ വാരിസിന്റെ കീഴിൽ, ആയിരക്കണക്കിന് വിദ്യാർത്ഥികൾ പ്രയോജനപ്പെടുത്തി കൊണ്ടിരിക്കുന്ന ഓഡിയോ പഠന ക്ലാസുകളാണ് ഈ പോഡ്കാസ്റ്റ് സീരീസിൽ ഉൾക്കൊള്ളിച്ചിരിക്കുന്നത്.</div></div><p>&nbsp;</p>]]>
+    </itunes:summary>
+    <!-- podcats audio as items -->
+    <!-- ==================================================================================== -->
+    <!-- BEGIN: chapter 0 - introduction -->
+    <item>
+      <title>0. ആമുഖം - വെളിച്ചം ഖുർആൻ ദർസ്</title>
+      <description>
+        <![CDATA[]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/0-intro/0-vqd-podcast-intro.mp3" type="audio/mp3" length="563"/>
+      <itunes:duration>563</itunes:duration>
+      <pubDate>Wednesday, 27 Mar ‎2019 ‏‎12:00:00 GMT</pubDate>
+      <guid>0-vqd-podcast-intro</guid>
+    </item>
+    <!--  -->
+    <item>
+      <title>1.ഖുർആൻ പഠനത്തിനു ഒരു ആമുഖം</title>
+      <description>
+        <![CDATA[<p>Velicham Qur'an Dars series</p><p>ഖുർആനുമായി ബന്ധപ്പെട്ട അടിസ്ഥാന വിശയങ്ങളാണ് ആമുഖമായി ഒരുക്കിയിരിക്കുന്ന ആദ്യ ക്ലാസുകൾ. ശ്രദ്ധയോടെ കേൾക്കുമല്ലോ.</p><p>ഖുർആൻ ക്ലാസ് #1</p><p>ഖുർആൻ പഠനത്തിന് ഒരാമുഖം (introduction to Qur'an study)</p><p>🔸Admin Team Velicham Onlive🔸</p>]]>
+      </description>
+      <enclosure url="https://velicham-onlive.github.io/audios/0-intro/1-vqd-podcast-intro.mp3" type="audio/mp3" length="614"/>
```

Comment:
@faisalmohd83 Please change all values of length attributes.
NB: Length should be the size of the file in bytes.

Reply:
OK